docs/src/domains/cdn/cloudflare.md

```diff
@@ -1,69 +1,52 @@
 ---
-title: "Cloudflare configuration"
+title: "Configure your Cloudflare CDN"
 weight: 2
 sidebarTitle: "Cloudflare"
 toc: false
 aliases:
   - "/golive/steps/cloudflare.html"
 ---
 
-Verify your registrar supports [`CNAME` records for apex domains](../steps/dns.md#handling-apex-domains).
-This solves the problem of being able to point an apex domain such as `example.com`
-to a domain name (using a `CNAME` record) rather than an IP address (using an A record).
-CloudFlare offers [`CNAME` flattening as an alternative](https://blog.cloudflare.com/introducing-cname-flattening-rfc-compliant-cnames-at-a-domains-root/).
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
 
-To correctly point DNS to your Platform.sh project,
-you need the [target for your Production environment](../../domains/steps/_index.md#2-get-the-target-for-your-project).
-
-Assuming that you are using both a `www.` subdomain and the bare domain,
-point both of those DNS entries to the same place.
-Whether you choose the bare domain version or the `www` subdomain doesn't make any practical difference,
-as they both reach Platform.sh and are handled correctly.
+By default, a [Fastly CDN](fastly.md) is included in Dedicated projects,
+but you can use a Cloudflare CDN instead, depending on your needs.
+Self-Service Grid plans don't automatically include a CDN, 
+but you can set up one at any time.
 
 {{% disable-cache CDN="Cloudflare" %}}
 
-## Enable "Full SSL" option in the Cloudflare admin
+## Enable Full (strict) SSL
 
-Cloudflare also makes it possible to use their free TLS/SSL service to secure your site via HTTPS,
-while also being behind their CDN if you so choose.
-If you decide to use Cloudflare's CDN functionality in addition to their DNS service,
-you should be sure to choose the **Full SSL** option in the Cloudflare admin.
+For enhanced security, you can enforce HTTPS communication between your visitors and Cloudflare,
+and between Cloudflare and your Platform.sh origin server.   
 
-This means that traffic to your site is encrypted from the client (browser) to Cloudflare's servers using their certificate,
-and also between Cloudflare's servers and your project hosting here at Platform.sh,
-mostly like using your project's Let's Encrypt certificate.
+To do so, from your Cloudflare account, [enable Full (strict) SSL]https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/full/.
+All HTTP requests are then automatically redirected to HTTPS.
 
-```text
-# Cloudflare's Full SSL option
-          https                       https
-User <---------------> Cloudflare <-------------> Platform.sh
-```
+## Renew your TLS certificates
```

Comment:
I couldn't verify the information in this section. I updated it by checking if the settings mentioned were still available in the Cloudflare interface, but I honestly have no idea if this is up-to-date or actually needed 🤷‍♀️ . Does it ring a bell?